ecommerce/permissions.py

```diff
@@ -31,3 +32,18 @@ def has_permission(self, request, view):
                 request.data,
             )
             return False
+
+
+class IsLoggedInUser(BasePermission):
+    """
+    Confirms that the username in the request body is the same as the logged in user's.
+    """
+
+    def has_permission(self, request, view):
+        """
+        Returns true if the username in the request body matches the logged in user.
+        """
+        try:
+            return request.data['username'] == get_social_username(request.user)
```

Comment:
I think the the username in the request is always the local username and not the edX one. 
Am I missing something here?